erde/cli/repl.lua

```diff
@@ -65,8 +71,12 @@ local function repl()
 
     if not ok then
       print(lib.rewrite(result))
-    elseif result ~= nil then
-      print(result)
+    elseif result.n > 0 then
+      for i = 1, result.n do
+        -- explicitly call `tostring()` so nils are also stringified
+        result[i] = tostring(result[i])
+      end
+      print(unpack(result))
```

Comment:
changed this to mutate the `result` table directly, instead of introducing another intermediate variable